README.md

```diff
@@ -15,6 +15,7 @@ PORT=3111
 REACT_APP_SHOW_NOTIFICATIONS=true
 REACT_APP_AUTHENTICATION_HOST=http://localhost:3001
 REACT_APP_H5P_INTEGRATION_URL=
+REACT_APP_VERSION=latest-dev
```

Comment:
Does it make sense for these to be auto-supplied? Can't we inject e.g. the latest git sha by default?
Otherwise I guess it's fine

Reply:
We can, although it is not as easy and assumes there is a .git repo present. If you just copied the code, there is no versioning info available. And this is for local development only ... we could replace if by `dev-local`

Reply:
I see, I think it's fine then